docs/wiki/getting-started/hardware/esc.md

```diff
@@ -0,0 +1,121 @@
+# ESC
+
+An ESC is the Electronic Speed Controller which supplies power to the craft's motors.
+
+- ESCs can support brushed or brushless motors, although brushed motors tend to only be seen in older, very lightweight quadcopters.
+- The ESC takes motor commands from the Flight Controller and outputs pulses of power to accelerate or decelerate each motor.
+- Dshot is the preferred protocol for the FC to control ESCs. Dshot variants are named by update speed - Dshot150, Dshot300 and Dshot600.
+- Older ESCs may use alternative protocols like PWM, Multishot, Oneshot, and Proshot. Only Dshot supports the telemetry needed for RPM filtering.
```

Comment:
Maybe:
Dshot is the preferred, digital, single-wire protocol for communication between the ESC and the FC.